ui/modules/dashboard/DashboardTransverse.tsx

```diff
@@ -168,11 +168,28 @@ const DashboardTransverse = () => {
           Aperçu des données de l’apprentissage
         </Heading>
         <Text fontSize={14} mt="8">
-          Ces chiffres reflètent partiellement les effectifs de l’apprentissage
+          Ces chiffres reflètent partiellement les effectifs de l’apprentissage&nbsp;:
           <Text as="span" fontWeight="bold">
             {auth.organisation.type === "TETE_DE_RESEAU" && " dans votre réseau"}
-          </Text>{" "}
-          : une partie des organismes de formation ne transmettent pas encore leurs données au tableau de bord.
+          </Text>
+          &nbsp;: une partie des organismes de formation en apprentissage ne transmettent pas encore leurs données au
+          tableau de bord (voir carte “Taux de couverture” ci-dessous). Ces chiffres reflètent partiellement les
+          effectifs de l’apprentissage
+        </Text>
+        <Text fontSize={14} mt="4">
+          Le{" "}
+          <strong>
+            {effectifsFilters.date.toLocaleDateString(undefined, {
+              month: "long",
+              day: "numeric",
+              year: "numeric",
+            })}
```

Comment:
Tu as le helper `formatDateDayMonthYear()` qui fait la même chose si tu veux l'utiliser (mais en forçant fr-FR).
Le undefined fait qu'on s'adapte à la locale du navigateur ?

Reply:
Oui ! C'est normalement "mieux" mais peut-être overkill sur un produit en fr-fr